crates/wasm-metadata/src/lib.rs

```diff
@@ -444,7 +453,11 @@ impl Metadata {
     fn display(&self, f: &mut fmt::Formatter, indent: usize) -> fmt::Result {
         let spaces = std::iter::repeat(" ").take(indent).collect::<String>();
         match self {
-            Metadata::Module { name, producers } => {
+            Metadata::Module {
+                name,
+                producers,
+                range: _,
```

Comment:
A better idiom for unused fields would be `Module { name, producer, .. }` which allows all unnamed fields to be unbound
```suggestion
                ..
```